web/templates/docs/publish.html.eex

```diff
@@ -1,5 +1,7 @@
-<h2>Publishing a package</h2>
-
+<div class="container page">
+<h2 class="page-title">
+  Publishing a package
+</h2>
```

Comment:
Not anymore, and I'm also moving the main template to the main layout, but I need to setup a variable for that in  templates we need to change and add an aditional class.